### PR TITLE
Sync fix

### PR DESCRIPTION
**Pull request by FilipLeitner**
- add geom change event listener for features added to source after the layer is pulled from layman
- Updating the geometry is not possible without the change of  [gemoetryName](https://openlayers.org/en/latest/apidoc/module-ol_Feature-Feature.html#getGeometryName) and [id](https://openlayers.org/en/latest/apidoc/module-ol_Feature-Feature.html#getId) that needs to match the one in layman

will be also neccessary to port to 2.x

### CODE REVIEW

**Comment by raitisbe** [comment]
Strange, i didnt have any problems with updating geometry. Is this something speciffic in the latest layman?

**Comment by raitisbe** [comment]
I guess I was always reloading the page after drawing new features, so didnt notice

**Comment by FilipLeitner** [comment]
> Strange, i didnt have any problems with updating geometry. Is this something speciffic in the latest layman?

Most likely yes. Wasnt looking into laymans code tho

**Comment by raitisbe** [comment]
No it was lack in my code. Layman wasnt to blame
